framework/src/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -287,6 +309,38 @@ protected static File extractFileArg(final String argumentName, final File alter
         return actualArgs;
     }
 
+    /**
+     * Remove the -processorpath options and their arguments from args.
+     * Return the last argument.
+     *
+     * @param args a list of arguments to extract from
+     * @return the arguments that should be put on the processorpath when calling javac.jar
+     */
+    protected static List<String> extractPpOpts(final List<String> args) {
+        List<String> actualArgs = new ArrayList<String>();
+
+        String path = null;
+
+        int i = 0;
+        while (i < args.size()) {
+
+            if (args.get(i).equals("-processorpath")) {
+                if (args.size() > i) {
```

Comment:
You are right.  I copied that from the extractCpOpts routine, which contained this bug.  I will fix both routines.  Thanks!